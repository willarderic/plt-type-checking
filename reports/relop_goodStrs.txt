TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: GT  LEXEME: >						TOKEN: GT        LEXEME: >
REL_OP -> >							REL_OP -> >
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"					TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"					TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: GE  LEXEME: >=						TOKEN: GE        LEXEME: >=
REL_OP -> >=							REL_OP -> >=
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"					TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"					TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: LT  LEXEME: <						TOKEN: LT        LEXEME: <
REL_OP -> <							REL_OP -> <
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"					TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"					TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: LE  LEXEME: <=						TOKEN: LE        LEXEME: <=
REL_OP -> <=							REL_OP -> <=
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"					TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"					TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: EQ  LEXEME: =						TOKEN: EQ        LEXEME: =
REL_OP -> =							REL_OP -> =
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"					TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"					TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: NE  LEXEME: /=						TOKEN: NE        LEXEME: /=
REL_OP -> /=							REL_OP -> /=
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"					TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"					TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR_LIST -> EXPR						EXPR_LIST -> EXPR
EXPR_LIST -> EXPR EXPR_LIST					EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST					EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST					EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST					EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST					EXPR_LIST -> EXPR EXPR_LIST
PARENTHESIZED_EXPR -> EXPR_LIST					PARENTHESIZED_EXPR -> EXPR_LIST
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR							START -> EXPR

---- Completed parsing ----					---- Completed parsing ----

