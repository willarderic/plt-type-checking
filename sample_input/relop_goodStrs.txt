( ; one big expr list to check all the relops with str args
(> "5" "6")
(>= "5" "6")
(< "5" "6")
(<= "5" "6")
(= "5" "6")
(/= "5" "6")
)
