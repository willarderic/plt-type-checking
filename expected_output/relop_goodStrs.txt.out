TOKEN: LPAREN    LEXEME: (
TOKEN: LPAREN    LEXEME: (
TOKEN: GT        LEXEME: >
REL_OP -> >
BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN    LEXEME: (
TOKEN: GE        LEXEME: >=
REL_OP -> >=
BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN    LEXEME: (
TOKEN: LT        LEXEME: <
REL_OP -> <
BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN    LEXEME: (
TOKEN: LE        LEXEME: <=
REL_OP -> <=
BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN    LEXEME: (
TOKEN: EQ        LEXEME: =
REL_OP -> =
BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN    LEXEME: (
TOKEN: NE        LEXEME: /=
REL_OP -> /=
BIN_OP -> REL_OP
TOKEN: STRCONST  LEXEME: "5"
CONST -> STRCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "6"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: RPAREN    LEXEME: )
EXPR_LIST -> EXPR
EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST
EXPR_LIST -> EXPR EXPR_LIST
PARENTHESIZED_EXPR -> EXPR_LIST
EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR

---- Completed parsing ----

